setup.cfg

```diff
@@ -0,0 +1,27 @@
+[metadata]
+name = git-filter-repo
+url = https://github.com/newren/git-filter-repo
+project_urls = 
+    Source = https://github.com/newren/git-filter-repo
+    Issues = https://github.com/newren/git-filter-repo/issues/
+description = git filter-branch replacement
+version = 2.25.0
+long_description = file: README.md
+author = Elijah Newren
+classifiers =
+    Development Status :: 3 - Alpha
+    Operating System :: OS Independent
+    Programming Language :: Python
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
```

Comment:
What's the typical use and meaning here?  Do folks list the oldest known working version of python and all newer versions?  Does each version listed mean it has been tested with that version?

Most of my testing has been on 3.5.  I used 3.4 for a little while on one box, and have 3.6 on some boxes.  I haven't actually tested newer versions, though I don't have any reason to believe they would have any issues.

Reply:
"Typically" is a tricky question :) -- small numbers of these change some things on PyPI.org -- but, roughly, what they usually mean is "this is a supported version".

Often "supported" corresponds with "tested", but not always.

Authors have different takes on what to put there -- I can tell you me as a maintainer, I generally run the test suite under all the versions I put here, but that that's over abundance of precaution -- in general, if you support some version, I think it's still safe to put all later versions on here.

(It's a quite minor concern though, these classifiers are purely informational, and you can imagine users therefore don't put a lot of implicit trust in them)

But based on what you said about your testing I'd probably personally recommend putting 3.5 and all later versions on.